bundles/net.tourbook/src/net/tourbook/tour/Cadence.java

```diff
@@ -0,0 +1,55 @@
+package net.tourbook.tour;
+
+import net.tourbook.Messages;
+
+public enum Cadence {
+
+   NONE(0), RPM(1), SPM(2), INVALID(255);
+
+   protected int value;
+   private String label;
+
+   private Cadence(final int value) {
+      this.value = value;
+
+      switch (value) {
+      case 0:
+         label = Messages.Enum_Cadence_None;
+         break;
+      case 1:
+         label = Messages.Enum_Cadence_Rpm;
+         break;
+      case 2:
+         label = Messages.Enum_Cadence_Spm;
+         break;
+      case 255:
+         label = Messages.Enum_Cadence_Invalid;
+         break;
+      }
+
+   }
+
+   public static Cadence getByValue(final int value) {
+
+      for (final Cadence type : Cadence.values()) {
+         if (value == type.value) {
+            return type;
+         }
+      }
+
+      return Cadence.INVALID;
+   }
```

Comment:
The cadence values are saved as integer values which makes it not very easy when viewing the xml file. 

There are already some helper functions to read/write enum values as text:

net.tourbook.common.util.Util.getStateEnum(...)
net.tourbook.common.util.Util.setStateEnum(...)